### ISSUE TITLE
Strip feeder setup: Use offsets for more accurate pickup

### ISSUE THREAD

**Issue by FlyingLotus1983** [TinWhiskers/openpnp#5]
@vonnieda could you elaborate on this one a bit?

**Comment by vonnieda** [comment]
@FlyingLotus1983 Yep, right now we just get the start and end hole and then pick blindly along that path. For more accurate pickup we could look at the holes as we go to correct any weirdness along the way. I think this is less useful, though, with working bottom vision. Bottom vision takes care of the offset better.

**Comment by FlyingLotus1983** [comment]
OK, I updated the description.  Personally, I think the best way to do make use of a [RANSAC](https://en.wikipedia.org/wiki/RANSAC) function after capturing all of the candidate circle matches.  That can help filter out detected points that clearly can't be part of the tape.  It also helps in that it returns a line that most closely represents the true direction of the tape (the least-squares method means that all measured dots have an influence on the line position and slope, not just the start and end detected points.

**Comment by vonnieda** [comment]
Yep, if we find we need better strip following accuracy that would work well.
